src/documentation/0003-node-installation/index.md

```diff
@@ -1,23 +1,17 @@
 ---
 title: How to install Node.js
 description: 'How you can install Node.js on your system: a package manager, the official website installer or nvm'
-authors: flaviocopes, ZYSzys, ollelauribostrom, MylesBorins, fhemberger, LaRuaNa, ahmadawais
+authors: flaviocopes, ZYSzys, ollelauribostrom, MylesBorins, fhemberger, LaRuaNa, ahmadawais, bhalverson
 section: Quick Start
 category: learn
 ---
 
 Node.js can be installed in different ways. This post highlights the most common and convenient ones.
 
-Official packages for all the major platforms are available at <https://nodejs.org/en/download/>.
+Official packages for all the major platforms are available at <https://nodejs.dev/download/>.
 
 One very convenient way to install Node.js is through a package manager. In this case, every operating system has its own.
 
-On macOS, [Homebrew](https://brew.sh/) is the de-facto standard, and - once installed - allows you to install Node.js very easily, by running this command in the CLI:
-
-```bash
-brew install node
-```
-
 Other package managers for Linux and Windows are listed in <https://nodejs.org/en/download/package-manager/>
```

Comment:
Since we removed Homebrew, maybe we should add Mac here too.

```suggestion
Other package managers for MacOS, Linux, and Windows are listed in <https://nodejs.org/en/download/package-manager/>
```